### ISSUE TITLE
no drop down help in Jupyter notebook

### ISSUE THREAD

**Issue by stilist80** [ipython/ipython#9741]
Hello, I am trying to find out why my Jupyter notebook does not show drop down help when the cursor is in the brackets. Already spend few hours and almost nothing. Please advice how to fix it as I see such feature quite useful for me. MacOS 10.10 and Safari. Thanks.

**Comment by takluyver** [comment]
Have you pressed shift-tab?

**Comment by stilist80** [comment]
of course I did, I also reinstalled and activated widgets and tried sample on Jupiter site. The developer console shows the following:

[Log] Default extension for cell metadata editing loaded. (main.min.js, line 24135)
[Log] Raw Cell Format toolbar preset loaded. (main.min.js, line 24221)
[Log] Slideshow extension for metadata editing loaded. (main.min.js, line 24269)
[Warning] actions jupyter-notebook:find-and-replace does not exist, still binding it in case it will be defined later... (main.min.js, line 29057)
[Log] Session: kernel_created (205ee6b1-1a34-4d36-8097-2b534624e6e6) (main.min.js, line 23423)
[Log] Starting WebSockets: ws://localhost:8889/api/kernels/52b1e6ba-4011-4719-9157-cc1f0f977a9c (main.min.js, line 22709)
[Error] Failed to load resource: the server responded with a status of 404 (Not Found) (jquery.min.map, line 0)
[Log] load_extensions ["jupyter-js-widgets/extension"] (main.min.js, line 12154)
[Log] Loading extension: jupyter-js-widgets/extension (main.min.js, line 12131)
[Log] Kernel: kernel_connected (52b1e6ba-4011-4719-9157-cc1f0f977a9c) (main.min.js, line 22356)
[Log] loaded widgets (extension.js, line 134)
[Log] Kernel: kernel_ready (52b1e6ba-4011-4719-9157-cc1f0f977a9c) (main.min.js, line 22356)
[Log] Widget backend and frontend versions are compatible (extension.js, line 32270)

I see no activity in error console when I put cursor in the bracket of print() function for instance. Should I install complete anaconda package or jupyter is enough to get it working by default?

**Comment by stilist80** [comment]
Firefox also doesn't show drop down menu. So it is on server side on jupyter. I got the list o when pressing Tab but no automatic dropdown for each function when cursor in the brackets. Please help.

**Comment by takluyver** [comment]
There's no automatic dropdown just by putting the cursor inside the brackets (I think there was in some really old version, but it changed quite some time ago). Now the tooltip is meant to appear when you press shift-tab.

**Comment by stilist80** [comment]
ok, I got it. Thanks for your time and have a nice day.

**Comment by pankajgaur583671** [comment]
> There's no automatic dropdown just by putting the cursor inside the brackets (I think there was in some really old version, but it changed quite some time ago). Now the tooltip is meant to appear when you press shift-tab.

Hey! I'm using Microsoft Edge to use Jupyter and nothing appears after pressing Shift+Tab. please help

**Comment by a13jndro** [comment]
> 

That worked for me. Thank you!

**Comment by dankristian** [comment]
I have the same problem. There is no drop-down in Jupiter.  Now I started a new course in Udemy and on another computer, where I install Python Anaconda Jupyter from scratch. there is still no drop-down menu. Like when I type "len" it should automatically have a drop-down menu, but there is nothing. I did try to ask in Udemy, but no solution or mostly no answer at all. The problem has haunted me from January to now in late August it is 7 months now.
